doc/rst/technotes/gpu.rst

```diff
@@ -63,7 +58,8 @@ there.  This may also be set using the ``CHPL_CUDA_ARCH`` environment variable.
 If you would like to view debugging information you can pass ``--verbose`` to
 your generated executable. This output will show the invocation of CUDA kernel
 calls along with various other interactions with the GPU such as memory
-operations.
+operations.  You may also use the GPUDiagnostics module to gather similar
```

Comment:
Can you add a link to the module documentation? Grepping `:ref:` in the technotes can lead to the right incantation for it.

Reply:
Updated; this is how you do it BTW: ``:mod:`GPUDiagnostics` ``